src/tracing/index.js

```diff
@@ -21,7 +21,7 @@ exports.init = function(_config) {
     require('./opentracing').init(config, automaticTracingEnabled);
 
     if (automaticTracingEnabled) {
-      require('./hook').init(config);
+      require('./cls');
```

Comment:
Why is it required if it doesn't need any initialization? As far as I can tell, requiring the module has no side effects, so this line can be removed.